portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -186,12 +186,55 @@ definition {
 	}
 
 	@description = "Verify that a link to a URL can be added to the Body Text"
-	@ignore = "true"
 	@priority = "5"
 	test CanSupportALinkToAURL {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131240 CanSupportALinkToAURL pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		Click(
+			key_titleName = "Link",
+			locator1 = "FormFields#TOOLBAR_ANY_ITEM_BUTTON");
+
+		Click(
```

Comment:
Hello @Jamersonze this invocation of Click macro and the next are not necessary as they're not setting the values, they're only clicking on this locators, thus I removed it.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/ab699f7fd710b0b25ca55956ef8bab175038f10b